src/test/java/com/codeofus/reservationservice/ParkingConsumerTest.java

```diff
@@ -0,0 +1,62 @@
+package com.codeofus.reservationservice;
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@AutoConfigureMockMvc
+public class ParkingConsumerTest extends BaseParkingConsumerIntegrationTest {
+    @Autowired
```

Comment:
Can you add a test that will actually succeed with serialization/deserialization of data from the parking service. I've had problems with Page<Something? in the past so just to be sure that it's resolved now.